src/partition/repart.c

```diff
@@ -6810,6 +6812,9 @@ static int help(void) {
                "                          signatures\n"
                "     --certificate=PATH   PEM certificate to use when generating verity\n"
                "                          roothash signatures\n"
+               "     --signing-engine=ENGINE\n"
+               "                          Specify OpenSSL engine to use when generating verity\n"
+               "                          roothash signatures\n"
```

Comment:
btw, what about --certificate=? does this also refer to a x509 cert on the engine if specified?

Reply:
Depends on the engine, so best left to the respective engine documentation

Reply:
Also here it was already being used via X.509 since the beginning